package.json

```diff
@@ -1,34 +1,37 @@
 {
-  "name": "sheikah-website",
-  "version": "1.0.0",
+  "name": "nuxt-app",
   "private": true,
   "scripts": {
+    "deploy": "gh-pages --dotfiles -d .output/public --repo git@github.com:witnet/sheikah-website.git --remote upstream",
```

Comment:
I think we can delete this script because we're deploying the website using a github actions. We can remove the npm package `gh-pages` as well